kong/clustering/rpc/json_rpc_v2.lua

```diff
@@ -32,7 +32,7 @@ function _M.new_error(id, code, msg)
     id = id,
     error = {
       code = code,
-      message = msg,
+      message = tostring(msg),
```

Comment:
If the function `new_error` expects a string/nil `err`, instead of call `tostring` every time, adding an assertion makes more sense, which helps us to expose potential bugs at the early stage.

Reply:
done